lib/actions.js

```diff
@@ -365,6 +365,19 @@ exports.viewport = function (width, height, done) {
 };
 
 /**
+ * Set the useragent.
+ *
+ * @param {String} useragent
+ * @param {Function} done
+ */
+
+exports.useragent = function(useragent, done) {
+  debug('.useragent() to ' + useragent);
```

Comment:
i think we should still listen for `this.child.once('useragent', ...)` since the IPC communication is async even if the useragent setter isn't.